confirm/confirm.go

```diff
@@ -33,3 +33,28 @@ func Ask(s string) bool {
 		}
 	}
 }
+
+// AskStrict asks the user for confirmation. A user must type the expected confirmation text and
+// then press enter.
+func AskStrict(s string, c string) bool {
+	reader := bufio.NewReader(os.Stdin)
+
+	fmt.Printf("%s: ", color.YellowString(s))
+
+	for {
+		response, err := reader.ReadString('\n')
+		if err != nil {
+			log.Fatal(err)
```

Comment:
I think the error [should be masked](https://github.com/giantswarm/fmt/blob/master/go/errors.md)